Examples/Swift/ShapeCollectionFeatureExample.swift

```diff
@@ -0,0 +1,52 @@
+import Mapbox;
+
+@objc(ShapeCollectionFeatureExample_Swift)
+
+class ShapeCollectionFeatureExample_Swift: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds)
+        mapView.styleURL = MGLStyle.lightStyleURL(withVersion: 9)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        
+        mapView.setCenter(CLLocationCoordinate2D(latitude:38.897435, longitude: -77.039679), zoomLevel: 12, animated: false)
+        
+        mapView.delegate = self
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        if let url = URL(string: "https://api.mapbox.com/datasets/v1/mapbox/cj004g2ay04vj2xls3oqdu2ou/features?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpemc0YWlpNzAwcXUyd21ldDV6OWpxMGwifQ.A92RQZpwUgtGtCmdSE4-ow"){
+            
+            let data = try! Data(contentsOf: url)
+            let feature = try! MGLShape(data: data, encoding: String.Encoding.utf8.rawValue) as! MGLShapeCollectionFeature
+            
+            // Create source and add it to the map style.
+            let source = MGLShapeSource(identifier: "transit", shape: feature, options: nil)
+            style.addSource(source)
+            
+            // Create station style layer.
+            let circleLayer = MGLCircleStyleLayer(identifier: "stations", source: source)
+            // Use a predicate to filter out non-points.
+            circleLayer.predicate = NSPredicate(format: "TYPE = %@", ["Station"])
```

Comment:
This predicate says `TYPE` must be the array containing `Station`. We don’t support setting feature properties to arrays, and the dataset always sets `TYPE` to a simple string anyways. So you probably mean either `TYPE = 'Station'` or `TYPE IN {'Station'}`. I’m guessing you arrived at this syntax because you meant to use `NSPredicate(format:argumentArray:)`, but `NSPredicate(format:)` is preferable, as explained above.